src/app/GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -298,6 +300,11 @@ private async Task LoadSortedTagsAsync()
         }
 
         private void ReloadCommitInfo()
```

Comment:
Why do we need this method? Right now, it looks like there are two different ways to initiate info reload sequence, but there's an ambiguity.

Reply:
Any reason not to do this:
```
        private void ReloadCommitInfo(CancellationToken cancellationToken = default)
        {
            if (cancellationToken == default)
            {
                _asyncLoadCancellation.Next();
            }

```

Reply:
What shall be the benefit of the slightly less readable and less explicit variant with default argument and extra comparison?
`ReloadCommitInfo` will not be extended with additional arguments. And it is not a public API.
What is unclear with the adjacent overloads?
```
        private void ReloadCommitInfo()
        {
            ReloadCommitInfo(_asyncLoadCancellation.Next());
        }

        private void ReloadCommitInfo(CancellationToken cancellationToken)
        {
```
?
BTW: Your suggestion becomes even longer because the result *must* be assigned to `cancellationToken`.

Reply:
> My main/only concern is possible confusion - which method to use.

I am going to consider this when extending an API.